src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -183,21 +183,21 @@ public function exportCommentAction(string $itemType, int $id,Request $request)
     {        
         $url=$request->getBaseUrl('_route').'/cftree/'.$itemType.'/'.$id;
         $repo = $this->getDoctrine()->getManager()->getRepository(Comment::class);
-        $comment_data=$repo->exportComments($itemType,$id);
         $rows=array();
         if($itemType=='item')
```

Comment:
You can refactor this to skip the repeated block, also you can ask for `$itemType` and remove the if condition.